src/scripts/clipperUI/componentBase.ts

```diff
@@ -105,6 +104,20 @@ export abstract class ComponentBase<TState, TProps> {
 				} else if (e.which === Constants.KeyCodes.tab) {
 					// Since they are using the keyboard, revert to the default value of the outline so it is visible
 					element.style.outlineStyle = "";
+				} else if (e.which === Constants.KeyCodes.up) {
+					if (element.hasAttribute("data-ariaSet")) {
+							let nextPosInSet = parseInt(element.getAttribute("aria-posinset"), 10) - 1;
+							const buttons = document.querySelectorAll("a[data-ariaSet=" + ariaSet.toString() + "]");
```

Comment:
Why is toString needed?

Reply:
Also this will throw an exception if ariaSet is undefined (ariaSet is an optional parameter)